src/main/java/com/project/Investment/App/util/math/BigFunctions.java

```diff
@@ -0,0 +1,55 @@
+package com.project.Investment.App.util.math;
+
+import java.math.BigDecimal;
+import java.math.MathContext;
+import java.math.RoundingMode;
+
+public class BigFunctions {
+
+    private static final MathContext mc = new MathContext(6);
+
+    public static Double round(Double value, int places) {
+        if (places < 0) throw new IllegalArgumentException();
```

Comment:
It's a bad practice to throw exception without message

Reply:
Done